src/agent/debuglet.ts

```diff
@@ -637,6 +661,14 @@ export class Debuglet extends EventEmitter {
     return (await metadata.instance('attributes/cluster-name')).data as string;
   }
 
+  static async getRegion(): Promise<string> {
+    if (process.env.FUNCTION_REGION) {
+      return process.env.FUNCTION_REGION;
+    }
+    const segments = ((await metadata.instance('region')) as string).split('/');
```

Comment:
Can you include a reference to the expected format of the region string?

Reply:
There is no such a reference for the format of the string. I'm adding an example I saw when deployed to the real GCF environment.